kibana/docker/Makefile

```diff
@@ -83,7 +83,8 @@ venv: requirements.txt
 	touch venv
 
 # Generate the Dockerfiles from Jinja2 templates.
-dockerfile: venv templates/Dockerfile.j2
+#dockerfile: venv templates/Dockerfile.j2
```

Comment:
won't we need Dockerfile.j2 for grabbing the right template?

Reply:
Check  line #94......